db/migrate/20190313142051_set_default_stock_location_on_shipments.rb

```diff
@@ -0,0 +1,8 @@
+class SetDefaultStockLocationOnShipments < ActiveRecord::Migration
+  def up
+    if Spree::Shipment.where('stock_location_id IS NULL').count > 0
+      location = DefaultStockLocation.find_or_create
```

Comment:
I would have gone the other way, defining `StockLocation` and `Shipment` here to be independent of code changes. But I'm happy to go with this for now.